README.md

```diff
@@ -11,6 +11,6 @@ Howto
 ```config
 method.insert = cfg.pushover.userkey, private|const|string, (cat,"PUSHOVER_USERKEY")
 method.insert = cfg.pushover.apitoken, private|const|string, (cat,"PUSHOVER_TOKEN")
-system.method.set_key = event.download.finished,notify_me,"execute=~/path/to/rtorrent-pushover.sh,$d.get_name="
+system.method.set_key = event.download.finished,notify_me,"execute=~/path/to/rtorrent-pushover.sh,$d.get_name=$cfg.pushover.userkey=,$cfg.pushover.apitoken="
```

Comment:
Nice find. I haven't used the multi-user part so this hasn't been a problem for me..
If I'm not mistaken you're missing a comma here between get_name & userkey?
$d.get_name=`,`$cfg.pushover.userkey=

If you fix that I can merge this!